src/plugins/intel_cpu/src/emitters/README.md

```diff
@@ -0,0 +1,112 @@
+
+JIT emitters are inherited from `jit_emitter` base class. The base class implementation depends on architecture:
+ * X64: [jit_emitter.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_emitter.hpp)
+ * ARM64: [jit_emitter.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.hpp)
+
+### Class diagram
+JIT emitters should be inherited from `jit_emitter` base class and it's usage should be added in `jit_uni_eltwise_generic` ARM64 kernel. 
+```mermaid
+classDiagram
+jit_kernel *-- jit_operation_emitter
+jit_operation_emitter --|> jit_emitter
+jit_emitter --|> Emitter
+```
+
+### Sequential diagram
+
+```mermaid
+sequenceDiagram
+    participant jit_kernel
+    participant jit_operation_emitter
+    participant jit_emitter
+    
+    jit_kernel->>jit_kernel: preamble
+
+    Note left of jit_operation_emitter: preparation
+    jit_kernel->>jit_operation_emitter: get_supported_precisions
+    jit_operation_emitter->>jit_kernel: 
+
+    jit_kernel->>jit_operation_emitter: constructor
+    jit_operation_emitter->>jit_emitter: prepare_table
+    jit_emitter->>jit_operation_emitter: 
+    jit_operation_emitter->>jit_operation_emitter: register_table_entries
+    jit_operation_emitter->>jit_kernel: 
+    jit_operation_emitter->>jit_operation_emitter: create dependent emitter instance
+    jit_operation_emitter->>jit_kernel: 
+
+    jit_kernel->>jit_operation_emitter: get_inputs_count
+    jit_operation_emitter->>jit_kernel: 
+
+    jit_kernel->>jit_operation_emitter: get_aux_vecs_count
+    jit_operation_emitter->>jit_kernel: 
+
+    jit_kernel->>jit_operation_emitter: get_aux_gprs_count
+    jit_operation_emitter->>jit_kernel: 
+
+    Note left of jit_operation_emitter: code emission
+    jit_kernel->>jit_emitter: emit_code
+    jit_emitter->>jit_emitter: emitter_preamble
+    jit_emitter->>jit_operation_emitter: emit_impl
+    jit_operation_emitter->>jit_operation_emitter: emit_isa
+    jit_operation_emitter->>jit_emitter: 
+    jit_emitter->>jit_emitter: emitter_postamble
+    jit_emitter->>jit_kernel: 
+
+    Note left of jit_operation_emitter: data emission
+    jit_kernel->>jit_operation_emitter: emit_data
+    jit_operation_emitter->>jit_emitter: emit_data
+    jit_emitter->>jit_operation_emitter: 
+    jit_operation_emitter->>jit_operation_emitter: dependent emitter: emit_data 
+    jit_operation_emitter->>jit_kernel: 
+    
+    jit_kernel->>jit_kernel: postamble
+```
+
+## Debugging
+
+To start debugging JIT emitter's source code you should:
+ * generate targets with `cmake` options:
+   * `-DCMAKE_BUILD_TYPE=Debug`
+   * `-DTHREADING=SEQ`
+ * set breakpoint in ARM64 kernel `jit_uni_eltwise_kernel::operator()` method, line: `ker_(const_args, indexes);`
+
+JIT code doesn't have C++ source code, as result you need some additional steps to debug. Debugging options depends on used IDE:
```

Comment:
I would ask you to add details in that statement. Like describe that usage of JIT emitters/JIt kernels invokes 2 major stages: code generation and execution. To debug generation part it is enough to put brakepoint into generate methods, while debugging execution of generated code requires steps you already put there.

Reply:
done